astropy/wcs/wcsapi/high_level_api.py

```diff
@@ -249,7 +254,7 @@ def values_to_high_level_objects(*world_values, low_level_wcs):
 
     This function uses the information in ``wcs.world_axis_object_classes`` and
     ``wcs.world_axis_object_components`` to convert low level "values"
-    `~.Quantity` objects, to high level objects (such as `~.SkyCoord).
+    `~.Quantity` objects, to high level objects (such as `~.SkyCoord`).
```

Comment:
p.s. The ref syntax so weird. Why `~.`?

Reply:
The ~ makes it display just the object name and not the fully qualified name, I.e. SkyCoord vs astropy.coordinates.SkyCoord

Reply:
Yeah but I thought it should be like `~astropy.units.Quantity`; where did the `astropy.units` go?

Reply:
Oh, you can also do drop things in front of the . and it will find the closest match.

Reply:
Hmm explicit is better than implicit but ok, thanks for the clarification!